### DISCUSSION TITLE
Sort Armor Results Display by Multiple columns

### DISCUSSION THREAD

**Discussion by rmonroe041** [Mijago/D2ArmorPicker#186] [Ideas]
When running the D2AP, can you ADD/CREATE the option to enable to **SORT** the 'Results Display' by **MULTIPLE** selected columns?
Example: "sort by (1)**Resilience** (highest to lowest), then (2)**Recovery** (highest to lowest), Then (3)**Discipline** (highest to lowest)"

**Comment by TheMojoDojo** [comment]
Hiya! Right now it's impossible for this feature to be implemented as it's not a feature of the table being used to show the results,  however, if we redesign the results tab we will take this into consideration
